### PR TITLE
Mmt 2034

### PR DESCRIPTION
**Pull request by jdf-eos**
Hopefully, this is the last round of bug fixes for the Rails 5.1 baseline of MMT.

### CODE REVIEW

File: app/controllers/conversions_controller.rb [hunk 1]
```diff
@@ -14,14 +14,15 @@ def convert
...
       @tempfile
     end
 
-    response = OgreClient.convert_shapefile(params)
+    response = OgreClient.convert_shapefile(params.permit!.to_h)
+    
 
     # render(json: current_user.to_fileupload(name, style), content_type: request.format)
-    render text: response.body, content_type: request.format
+    render :json => response.body
```
Comment (ryanmiller-1) [inline app/controllers/conversions_controller.rb]: I think render json: response.body is the preferred syntax.

Comment (jdf-eos) [reply app/controllers/conversions_controller.rb]: This produces a syntax error:
SyntaxError
/Users/jdfarley/Work/mmt/app/controllers/conversions_controller.rb:26: syntax error, unexpected tIDENTIFIER, expecting keyword_end
    render :json response.body
                 ^~~~~~~~
Extracted source (around line #26):

It took several tries to find "render :json => response.body" worked correctly.   render has gone through some changes in Rails 5

Thoughts?

Comment (ryanmiller-1) [reply app/controllers/conversions_controller.rb]: Does 'json:' cause a syntax error?  json: response.body should be interpreted as a hash here.

Comment (jdf-eos) [reply app/controllers/conversions_controller.rb]: Sorry,  mis-typed!  render JSON: response.body  does work.  I'll incorporate this change.

**Pull request merged**